keras_nlp/layers/rotary_embedding.py

```diff
@@ -16,9 +16,47 @@
 
 
 class RotaryEmbedding(keras.layers.Layer):
-    def __init__(self, rotary_percentage, max_wavelength=10000):
+    """Rotary positional encoding layer.
+
+        Tbjs layer encodes absolute positional information with rotation matrix and naturally
```

Comment:
tbjs -> This. The alignment this whole docstring looks wrong, just four space in should do it.

Also please make sure to check everything besides links is <= 80 characters.

Reply:
Is there any automated way to work this out?
I make sure to run `./shell/format.sh` everytime